Application/Telegram/Handlers/UpdateCallbackQuery.cs

```diff
@@ -33,13 +33,13 @@ public static async Task Handle(Update update, TelegramBotClient telegraBotClien
                     case "New Booking":
                         UserStateStorage.UserStateUpdate(update.CallbackQuery.From.Id, UserState.NewBookingIsSelected);
                         await new ProvideButtons(telegraBotClient).SendAsync(
-                            update.CallbackQuery, new List<string>() { "Next", "BACKStartingProcess" }, 1);
+                            update.CallbackQuery, new List<string>() { "Next", "BACK" }, $"You choose: { update.CallbackQuery.Data} \n Press Buttonn", 1, "StartingProcess");
 
                         return;
                     case "New Vacation":
                         UserStateStorage.UserStateUpdate(update.CallbackQuery.From.Id, UserState.NewVacationIsSelected);
                         await new ProvideButtons(telegraBotClient).SendAsync(
-                            update.CallbackQuery, new List<string>() { "Next", "BACKStartingProcess" }, 1);
+                            update.CallbackQuery, new List<string>() { "Next", "BACK" }, $"You choose: {update.CallbackQuery.Data} \n Press Button", 1, "StartingProcess");
```

Comment:
Separate this parameters by new line, it is too long string

Reply:
done